cmd/thanos/bucket.go

```diff
@@ -335,7 +335,8 @@ func registerBucketWeb(m map[string]setupFunc, root *kingpin.CmdClause, name str
 			prober.NewInstrumentation(comp, logger, extprom.WrapRegistererWithPrefix("thanos_", reg)),
 		)
 
-		srv := httpserver.New(logger, reg, comp, httpProbe,
+		srv := httpserver.New(logger, reg, comp, "/",
```

Comment:
Do we want `webRoutePrefix` for all components, instead of just using `/` here? cc @GiedriusS @bwplotka

Reply:
I am not sure if it is required for all components. 
@bwplotka any inputs here?